hs-bindgen/src/HsBindgen/Backend/TH.hs

```diff
@@ -42,6 +43,7 @@ mkGlobal =  \case
       Applicative_seq      -> '(<*>)
       Monad_return         -> 'return
       Monad_seq            -> '(>>)
+      Show_Show            -> ''Show
```

Comment:
I'd name this `Show_class` (and rename `Storable_Storable` to `Storable_class`).